src/ui/component/viewers/videoViewer/view.jsx

```diff
@@ -6,6 +6,24 @@ import 'video.js/dist/video-js.css';
 import isUserTyping from 'util/detect-typing';
 
 const SPACE_BAR_KEYCODE = 32;
+const SMALL_F_KEYCODE = 70;
+const SMALL_M_KEYCODE = 77;
+const ARROW_LEFT_KEYCODE = 37;
+const ARROW_UP_KEYCODE = 38;
+const ARROW_RIGHT_KEYCODE = 39;
+const ARROW_DOWN_KEYCODE = 40;
+
+const FULLSCREEN_KEYCODE = SMALL_F_KEYCODE;
+const MUTE_KEYCODE = SMALL_M_KEYCODE;
+const VOLUME_INCREASE_KEYCODE = ARROW_UP_KEYCODE;
+const VOLUME_DECREASE_KEYCODE = ARROW_DOWN_KEYCODE;
```

Comment:
I don't think we need the volume change on up/down

Reply:
- removed the volume shortcuts
- updated the CHANGELOG